app/User.php

```diff
@@ -28,6 +28,18 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
-    protected $test = 'testtest';
-    protected $test2 = 'test2test2';
+    private $test = 'testzzzzzzzzzzzzz';
+    private $test2 = 'test2123';
+    private $test3 = 'tttt';
+    private $test4 = '12345876';
+    private $test5 = '12345876';
+
+    private function testComment() {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "return". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1763)